[2024-10-25T00:19:07.023+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-25T00:19:07.161+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Assignment8_ETL_DAG.copyToSessionChannel scheduled__2024-10-24T00:00:00+00:00 [queued]>
[2024-10-25T00:19:07.206+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Assignment8_ETL_DAG.copyToSessionChannel scheduled__2024-10-24T00:00:00+00:00 [queued]>
[2024-10-25T00:19:07.206+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-25T00:19:07.277+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): copyToSessionChannel> on 2024-10-24 00:00:00+00:00
[2024-10-25T00:19:07.397+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6397) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-25T00:19:07.342+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Assignment8_ETL_DAG', 'copyToSessionChannel', 'scheduled__2024-10-24T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/Assignment8_ETL_DAG.py', '--cfg-path', '/tmp/tmpg4znrnjh']
[2024-10-25T00:19:07.413+0000] {standard_task_runner.py:63} INFO - Started process 6415 to run task
[2024-10-25T00:19:07.413+0000] {standard_task_runner.py:91} INFO - Job 32: Subtask copyToSessionChannel
[2024-10-25T00:19:07.917+0000] {task_command.py:426} INFO - Running <TaskInstance: Assignment8_ETL_DAG.copyToSessionChannel scheduled__2024-10-24T00:00:00+00:00 [running]> on host d99256b0cb87
[2024-10-25T00:19:08.392+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Assignment8_ETL_DAG' AIRFLOW_CTX_TASK_ID='copyToSessionChannel' AIRFLOW_CTX_EXECUTION_DATE='2024-10-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-24T00:00:00+00:00'
[2024-10-25T00:19:08.400+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-25T00:19:08.406+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.4-linuxkit-x86_64-with-glibc2.36
[2024-10-25T00:19:08.412+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-10-25T00:19:12.614+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-10-25T00:19:12.627+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-25T00:19:12.667+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-25T00:19:12.752+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Assignment8_ETL_DAG, task_id=copyToSessionChannel, run_id=scheduled__2024-10-24T00:00:00+00:00, execution_date=20241024T000000, start_date=20241025T001907, end_date=20241025T001912
[2024-10-25T00:19:12.825+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-25T00:19:12.860+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-25T00:19:12.865+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
